invoicing/models.py

```diff
@@ -51,21 +71,35 @@ def get_amount(self):
     def get_percentage(self):
         return self.value if self.unit == InstallmentUnit.PERCENT else None
 
+    def get_corresponding_apartment_installment(self, apartment_data):
+        apartment_installment = ApartmentInstallment()
 
-class ApartmentInstallment(InstallmentBase):
-    apartment_reservation = models.ForeignKey(
-        ApartmentReservation,
-        verbose_name=_("apartment reservation"),
-        related_name="apartment_installments",
-        on_delete=models.PROTECT,
-    )
-    reference_number = models.CharField(
-        max_length=64, verbose_name=_("reference number"), blank=True
-    )
+        excluded_fields = ("id", "created_at")
+        for field_name in [
+            f.name
+            for f in InstallmentBase._meta.get_fields()
+            if f not in excluded_fields
+        ]:
```

Comment:
Not a big one, but this can be hard to read. Not a must-fix, but I would rather see the list of field names be created before the for-loop. This is however just a nitpick and _does not have to be fixed_.